teps/0091-trusted-resources.md

```diff
@@ -124,6 +124,8 @@ sign a given OCI image or other artifacts including binaries, scripts etc. This
 This proposal will introduce new verification into Tekton Pipelines' webhook to verify a
 Tekton Resource is tampered or not, and can indicate that the verification must occur.
 
+**Note:** For local resource, it will be verified first when applied to cluster, and when it is referred in taskrun/pipelinerun, the resource will be resolved and verified again to make sure it can still pass the verification.
```

Comment:
Clarity suggestion: 

Resources (`Task`, `Pipeline`) will be verified *both* when applied to the cluster *and* when referenced in `taskRun`/`pipelineRun`. When the `Run` is created, referenced resources will be resolved and verified again to confirm continued verification. This will prevent the `Run` of a resource that was verifiable when created but is no longer verifiable, perhaps due to key revocation, a security breach, or discovery of a new vulnerability since the time of the initial verification.

Reply:
Thank you!!!